docs_input/api/manipulation/joinrepeat/clone.rst

```diff
@@ -17,3 +17,8 @@ Examples
    :end-before: example-end clone-test-1
    :dedent:
 
+.. literalinclude:: ../../../../test/00_operators/clone_test.cu
+   :language: cpp
+   :start-after: example-begin clone-test-2
+   :end-before: example-end clone-test-2
+   :dedent:
```

Comment:
Do I need to make the same change to the `docs` directory as well?

Reply:
no, just docs_input